src/encoder/serverencoder/lobbyInfo.ts

```diff
@@ -0,0 +1,39 @@
+import BinaryReader from "../../binary/BinaryReader";
+import BinaryWriter from "../../binary/BinaryWriter";
+import { LobbyInfo } from "../../message/serverMessage";
+
+export const lobbyInfo = {
+  encode(data: LobbyInfo): Buffer {
+    const writer = new BinaryWriter();
+
+    const players = data.players;
+    const playerCount = players.length;
+
+    writer.writeString(data.adminId);
+    writer.writeInt32(playerCount);
+    for (let i = 0; i < playerCount; i++) {
+      writer.writeString(players[i].id);
+      writer.writeString(players[i].nickname);
+    }
```

Comment:
forEach 문으로 가독성을 향상시켜 볼까요?

```suggestion
   players.forEach((player) => {
      writer.writeString(player.id);
      writer.writeString(player.nickname);
    })
```